ambry-account/src/main/java/com/github/ambry/account/AccountServiceMetrics.java

```diff
@@ -102,6 +103,8 @@ public AccountServiceMetrics(MetricRegistry metricRegistry) {
         metricRegistry.counter(MetricRegistry.name(HelixAccountService.class, "AccountFetchFromAmbryServerErrorCount"));
     getAccountInconsistencyCount =
         metricRegistry.counter(MetricRegistry.name(CompositeAccountService.class, "GetAccountInconsistencyCount"));
+    onDemandContainerFetchCount =
+        metricRegistry.counter(MetricRegistry.name(MySqlAccountService.class, "onDemandContainerFetchCount"));
```

Comment:
Start name upper case for consistency.
Would you mind creating string constants for these names (I know most were already there).  It makes it safer for closed source to reference and avoid typos.